dd-java-agent/src/main/java/datadog/trace/bootstrap/BootstrapInitializationTelemetry.java

```diff
@@ -126,7 +135,31 @@ public void onFatalError(Throwable t) {
 
     @Override
     public void onError(String reasonCode) {
+      error = true;
       onPoint("library_entrypoint.error", "error_type:" + reasonCode);
+      setMetadata("error", reasonCode, mapResultClass(reasonCode));
+    }
+
+    @Override
+    public void setMetadata(String result, String resultReason, String resultClass) {
+      initMetaInfo("result", result);
+      initMetaInfo("result_reason", resultReason);
+      initMetaInfo("result_class", resultClass);
+    }
```

Comment:
I have my personal opinion, that it would be better to rearrange this code a bit...
You would need to change related places.
  ```suggestion
    @Override
    public void setMetadata(String result, String resultClass, String resultReason) {
      initMetaInfo("result", result);
      initMetaInfo("result_class", resultClass);
      initMetaInfo("result_reason", resultReason);
    }
```